content/how-to/account-management/account-management-faq/index.md

```diff
@@ -28,15 +28,14 @@ Customers can opt out by simply removing the account users.
 Yes, RBAC is available to all Rackspace customers.
 {{</ accordion >}}
 
-{{< accordion title="When I add a user login to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
+{{< accordion title="When I add a user login credential to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
 
-Yes, you can. For instructions, see [Use Role-Based Access Control
-(RBAC)](/support/how-to/managing-role-based-access-control-rbac).
+Yes, you can. For instructions, see [Use Role-Based Access Control (RBAC)](/support/how-to/use-role-based-access-control-rbac).
 {{</ accordion >}}
 
-{{< accordion title="How do I query the capability of a given user?" col="in" href="accordion20" >}}
+{{< accordion title="How can I find out a user's roles and responsibilities?" col="in" href="accordion20" >}}
 
-You can query the roles for a given user by using the [List global roles assigned to a user](https://docs.rackspace.com/docs/cloud-identity/v2/api-reference/role-operations/#list-global-roles-assigned-to-a-user) API operation described in the *Identity API Guide*. Additionally, you can view the roles that a user has through the [Cloud Control Panel](https://login.rackspace.com).
+Using the [List global roles assigned to a user](https://docs.rackspace.com/docs/cloud-identity/v2/api-reference/role-operations/#list-global-roles-assigned-to-a-user) API methods defined in the *Identity API Guide*. you can check about the roles for a certain user. Through the [Cloud Control Panel](https://login.rackspace.com), you can also see the roles that each user has.
```

Comment:
Keep the new links but revert to the original text and sentences that were around it. The new text is more confusing.